reverse_engineering/api.js

```diff
@@ -177,7 +169,7 @@ const handleFileData = filePath => {
 	return new Promise((resolve, reject) => {
 		fs.readFile(filePath, 'utf-8', (err, content) => {
 			if (err) {
-				reject(err);
+				reject(new Error(`handle file data error: ${err}`));
```

Comment:
Is there a reason to stringify the original error and lose the stack trace info? Just wonder about the added value of this change.

Reply:
sonar was complaining about it 😄 but I agree, that it doesn't make much sense to stringify the error, will change it back